cdk/src/components/hey-amplify-app.ts

```diff
@@ -255,39 +252,6 @@ export class HeyAmplifyApp extends Construct {
       })
     }
 
-    /**
-     * Set up alarms
-     */
-    const restartTopic = new sns.Topic(this, 'ContainerRestarts', {
-      displayName: 'ContainerRestarts',
-    })
-
-    const restartRuleTarget = new targets.SnsTopic(restartTopic, {
-      message: events.RuleTargetInput.fromText(
-        `Container restarted in ${process.env.CDK_DEFAULT_REGION as string}`
-      ),
-    })
-
-    /**
-     * Set up event rule for container restarts
-     */
-    const restartRule = new events.Rule(this, 'RestartRule', {
```

Comment:
What did you notice with this?

Reply:
we'll revisit this at a later time. The rule is currently not working as expected and was causing lint errors (due to the unused variable) after upgrading lint tooling